encrypted-media-respec.html

```diff
@@ -537,7 +537,18 @@
           </p>    
           <p>Time is intended to be consistent with Unix time.   
 		  </p>
-        </dd> 
+        </dd>
+        <dt id="user-agent-profile">User Agent Profile</dt>
+        <dd>
+          <p>
+            A User Agent on a given machine may support execution in a variety of different contexts or modes that are expected to behave independently in respect of page visible data.
```

Comment:
Include temporary state (and nits): "with respect to application-visible state and data."